content/faculty/partners.md

```diff
@@ -0,0 +1,69 @@
+title: Наши партнеры
+slug: partners
+category: faculty
+tags: [_nav]
+---
+
+##"ОМА"##
```

Comment:
How does this supposed to look like when rendered? Or it is just abuse of hash marks?
http://daringfireball.net/projects/markdown/basics